velox/common/base/RawVector.h

```diff
@@ -92,15 +92,15 @@ class raw_vector {
     return data_[index];
   }
 
-  T operator[](int32_t index) const {
+  const T& operator[](int32_t index) const {
     return data_[index];
   }
 
   void push_back(T x) {
     if (UNLIKELY(size_ >= capacity_)) {
       grow(size_ + 1);
     }
-    data_[size_++] = x;
+    data_[size_++] = std::move(x);
```

Comment:
There is no point (only wasting memory for no reason) to use `raw_vector` for types that is not trivially copyable.  Can you add a `static_assert(std::is_trivially_copyable_v<T>)` instead?

Reply:
Makes sense. I'll provide both a `static_assert` and a comment on the class itself to make it clear.